buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -31,30 +33,28 @@ object Versions {
     object Mod {
         const val mixin = "0.12.5+mixin.0.8.5"
         
-        const val minecraft = "1.20.2"
-        const val yarn = "$minecraft+build.4"
-        const val fabricLoader = "0.14.23"
+        const val minecraft = "1.20.4"
+        const val yarn = "$minecraft+build.1"
+        const val fabricLoader = "0.15.0"
         
-        const val architecuryLoom = "1.3.357"
-        const val architecturyPlugin = "3.4.146"
-        
-        const val loomVineflower = "1.11.0"
-    }
-    
-    object Forge {
-        const val forge = "${Mod.minecraft}-48.0.13"
-        const val burningwave = "12.63.0"
-    }
+        const val architecuryLoom = "1.4.367"
+        const val architecturyPlugin = "3.4.151"
+     }
+//
+//    object Forge {
+//        const val forge = "${Mod.minecraft}-48.0.13"
+//        const val burningwave = "12.63.0"
+//    }
     
     object Bukkit {
-        const val paper = "1.18.2-R0.1-SNAPSHOT"
-        const val paperLib = "1.0.5"
-        const val foliaLib = "0.2.5"
-        const val minecraft = "1.20.2"
-        const val reflectionRemapper = "0.1.0-SNAPSHOT"
-        const val paperDevBundle = "1.20.2-R0.1-SNAPSHOT"
-        const val runPaper = "2.2.0"
-        const val paperWeight = "1.5.6"
+        const val minecraft = "1.20.4"
```

Comment:
this PR has morphed into a 1.20.4 pr because 1.20.3 is basically dead (paper is unlikely to even release a 1.20.3 version). the reason that the NMS version is still v1_20_R3 is because the spigot api version has not changed between 1.20.3 and 1.20.4